### PR TITLE
feature(main): sealos merge fearure

### PR DESCRIPTION
**Pull request by cuisongliu**
Signed-off-by: cuisongliu <cuisongliu@qq.com>

### CODE REVIEW

File: pkg/image/merge.go [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	var entrypoints []string
+	var isRootfs bool
+	for i, oci := range ociList {
+		labels = maps.MergeMap(labels, oci.Config.Labels)
+		if oci.Config.Labels != nil && oci.Config.Labels[constants.ImageTypeKey] == string(v1beta1.RootfsImage) {
+			isRootfs = true
+		}
+		envs = maps.MergeMap(envs, maps.ListToMap(oci.Config.Env))
+		cmds = append(cmds, oci.Config.Cmd...)
+		if i == len(ociList)-1 {
```
Comment (cuisongliu) [inline pkg/image/merge.go:126]: @fanux It seems that Entrypoint doesn't have a better solution to merge?

File: cmd/sealos/cmd/merge.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+func newMergeCmd() *cobra.Command {
+	var options types.BuildOptions
+	var newImageName string
+	var mergeCmd = &cobra.Command{
+		Use:   "merge",
+		Short: "merge multiple images into one",
+		Long:  `sealos merge image1:latest image2:latest image3:latest ......`,
+		Example: `
+merge images:
+	sealos merge kubernetes:v1.19.9 mysql:5.7.0 redis:6.0.0 -t new:0.1.0
```
Comment (fengxsong) [inline cmd/sealos/cmd/merge.go:38]: `sealos merge x1 x2 x3 x4`, `x4` is the final image name.

Comment (cuisongliu) [reply cmd/sealos/cmd/merge.go]: Specifying parameters may be clearer.

File: pkg/image/merge.go [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+	for i, entrypoint := range entrypoints {
+		entrypoints[i] = "\"" + strings.ReplaceAll(entrypoint, "$", "\\$") + "\""
+	}
+	for i, cmd := range cmds {
+		cmds[i] = "\"" + strings.ReplaceAll(cmd, "$", "\\$") + "\""
+	}
+	t := template.New("")
+	t, err := t.Parse(
+		`FROM scratch
+MAINTAINER sealyun
```
Comment (fanux) [inline pkg/image/merge.go:146]: sealyun->labring

File: pkg/image/merge.go (hunk 1 above)
Comment (fanux) [reply pkg/image/merge.go:126]: why not same with CMD?

**Pull request merged**